proto/derivatives/derivatives.proto

```diff
@@ -56,7 +56,7 @@ message Position {
   ];
 }
 
-message Pool {
+message PoolParams {
```

Comment:
If PoolParams is used only in Params type, it might be better to define Pool message at inside of Params message,
but it is up to you.

Reply:
I don't see the merits of that style. Plus, we don't use it in the other modules.
So, I leave it as it is.